website/docs/docs/build/projects.md

```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
 
 For complete details on project configurations, see [dbt_project.yml](/reference/dbt_project.yml).
 
+## Project subdirectories
+
+The Project Subdirectory option in dbt Cloud allows you to specify a subdirectory within your git repository that dbt should use as the root directory for your project. This can be useful in situations where you have multiple dbt projects within a single repository, or when you want to organise your dbt project files into subdirectories for easier management.
+
+To use the Project Subdirectory option in dbt Cloud, follow these steps:
+
+1. **Click** the Settings dropdown on the top navigation menu and **Click** Account Settings
+<img width="379" alt="project-subdirectory-01" src="https://user-images.githubusercontent.com/114556261/228104923-591cd791-6f38-4a10-a983-7d85cd18f0b2.png">
```

Comment:
i can't see the image here. to add images:
- use <Lightbox src="/img/docs/your_image-name.jpg" title="Add your title here"/> 
- in that same branch, add your images probably in this [location](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/static/img/docs/building-a-dbt-project) of the docs repo (please use jpg)